data/arrow_test.go

```diff
@@ -290,6 +308,15 @@ func goldenDF() *data.Frame {
 	return df
 }
 
+func newField[V any](name string, ftype data.FieldType, vals []V) *data.Field {
```

Comment:
This is the one place we are using generics -- it is not really required, but sure simplifies writing this test function!